account_spread_cost_revenue/views/account_spread.xml

```diff
@@ -46,14 +47,26 @@
                                    attrs="{'invisible':[('invoice_type','not in',('out_invoice','in_refund'))]}"/>
                             <label for="debit_account_id" colspan="3" string="Expense account"
                                    attrs="{'invisible':[('invoice_type','not in',('in_invoice','out_refund'))]}"/>
-                            <span class="help-block" colspan="2"
-                                  attrs="{'invisible':[('invoice_type','not in',('out_invoice','in_refund'))]}">
-                                  The Balance Sheet account used for the spreading.<br/>This account is the counterpart of the account in the invoice line.
-                            </span>
-                            <span class="help-block" colspan="2"
-                                  attrs="{'invisible':[('invoice_type','not in',('in_invoice','out_refund'))]}">
-                                  The Expense account in the vendor bill line.<br/>Usually the same account of the vendor bill line.
-                            </span>
+                            <div attrs="{'invisible': [('use_invoice_line_account', '=', True)]}" colspan="2">
+                                <span class="help-block" colspan="2"
+                                      attrs="{'invisible':[('invoice_type','not in',('out_invoice','in_refund'))]}">
+                                      The Balance Sheet account in the vendor bill line.<br/>Usually the same account of the vendor bill line.
```

Comment:
I don't understand why you need to modify this part.
Anyway there's something wrong in this part, could you check? For example `out_invoice` is for sale invoices but here it is mentioned vendor bill...

Reply:
This is to make sure that the message is displayed according to the new use case.
If you think it is not necessary, I am ok to remove it. I think it is too complicated also.

Reply:
For me is useful to keep this part, as long as it is correct. But it seems wrong to me.
I'm checking in Runbot all the combinations. Terms like "invoices" and "vendor bills" are sometimes inverted. Terms like "counterpart" and "account" are sometimes inverted.  Would you please also check?

Reply:
Could be something like this:


```xml
                            <div attrs="{'invisible': [('use_invoice_line_account', '=', True)]}" colspan="2">
                                <span class="help-block" colspan="2"
                                      attrs="{'invisible':[('invoice_type','not in',('out_invoice','in_refund'))]}">
                                      The Balance Sheet account used for the spreading.<br/>This account is the counterpart of the account in the invoice line.
                                </span>
                                <span class="help-block" colspan="2"
                                      attrs="{'invisible':[('invoice_type','not in',('in_invoice','out_refund'))]}">
                                      The Expense account in the vendor bill line.<br/>Usually the same account of the vendor bill line.
                                </span>
                            </div>
                            <div attrs="{'invisible': [('use_invoice_line_account', '!=', True)]}" colspan="2">
                                <span class="help-block" colspan="2"
                                      attrs="{'invisible':[('invoice_type','not in',('out_invoice','in_refund'))]}">
                                      The Balance Sheet account.<br/>This is the account in the invoice line.
                                </span>
                                <span class="help-block" colspan="2"
                                      attrs="{'invisible':[('invoice_type','not in',('in_invoice','out_refund'))]}">
                                      The Expense account used for the spreading.<br/>This account is the counterpart of the account of the vendor bill line.
                                </span>
                            </div>
```
 
and

```xml

                            <div attrs="{'invisible': [('use_invoice_line_account', '=', True)]}" colspan="2">
                                <span class="help-block" colspan="2"
                                      attrs="{'invisible':[('invoice_type','not in',('out_invoice','in_refund'))]}">
                                      The Revenue account in the invoice line.<br/>Usually the same account of the invoice line.
                                </span>
                                <span class="help-block" colspan="2"
                                      attrs="{'invisible':[('invoice_type','not in',('in_invoice','out_refund'))]}">
                                      The Balance Sheet account used for the spreading.<br/>This account is the counterpart of the account in the vendor bill line.
                                </span>
                            </div>
                            <div attrs="{'invisible': [('use_invoice_line_account', '!=', True)]}" colspan="2">
                                <span class="help-block" colspan="2"
                                      attrs="{'invisible':[('invoice_type','not in',('out_invoice','in_refund'))]}">
                                      The Revenue account used for the spreading.<br/>This account is the counterpart of the account of the invoice line.
                                </span>
                                <span class="help-block" colspan="2"
                                      attrs="{'invisible':[('invoice_type','not in',('in_invoice','out_refund'))]}">
                                      The Balance Sheet account.<br/>This is the account in the vendor bill line.
                                </span>
                            </div>
```